plugin/trino-pinot/pom.xml

```diff
@@ -34,26 +34,6 @@
         </repository>
     </repositories>
 
-    <dependencyManagement>
```

Comment:
this section was added in https://github.com/trinodb/trino/pull/7627/files to avoid conflicting versions.

As long as enforcer doesn't complain about required upper bounds then it's good to remove.
I'd like @elonazoulay to confirm this once though.

Reply:
I checked, it doesn't complain (on the LTS branches it complains about zookeeper only; seems most of this section is long obsolete)